### ISSUE TITLE
Cannot install without existing package-lock.json

### ISSUE THREAD

**Issue by Fabrik19** [rafaelpernil2/ParallelPromises#29]
While trying to install this package without an existing package-lock the following error occurs:

```
npm ERR! code 1
npm ERR! path /home/[...]/node_modules/parallel-promises
npm ERR! command failed
npm ERR! command sh -c npx npm-force-resolutions
npm ERR! Error: ENOENT: no such file or directory, open './package-lock.json'
```

To recreate this error simply execute these commands:
```bash
rm -r node_modules/
rm package-lock.json
npm i 
```

**Comment by Fabrik19** [comment]
I has nothing to do if the package-lock exists. It is just not possible anymore to add this package.
This commit broke it: https://github.com/rafaelpernil2/ParallelPromises/commit/0526aa2d965a7ec57e27a954a96631c6d680b545

**Comment by rafaelpernil2** [comment]
Thanks for reporting the issue @Fabrik19 

I just fixed it, please try with 1.6.11

PR: https://github.com/rafaelpernil2/ParallelPromises/pull/30

**Comment by Fabrik19** [comment]
1.6.11 works :+1:
